x-pack/plugins/uptime/server/lib/requests/get_pings.ts

```diff
@@ -53,6 +53,21 @@ const REMOVE_NON_SUMMARY_BROWSER_CHECKS = {
   ],
 };
 
+const excludedLocationsClause = (excludedLocations: string[]) =>
```

Comment:
Makes me wonder why we don't allow more powerful filtering in this API? Future ER?